### ISSUE TITLE
Resize uBlock window [Unnecessary Feature]

### ISSUE THREAD

**Issue by davidhedlund** [uBlock-LLC/uBlock#1396]
![1](https://cloud.githubusercontent.com/assets/6416073/7702398/11272812-fe30-11e4-82ab-41b75fee47c8.png)
![2](https://cloud.githubusercontent.com/assets/6416073/7702400/116f3b0c-fe30-11e4-859b-e84cc61c5983.png)

To make everything visible for full screenshots instead of having to take one screenshot, scroll down, and take another:
- It would be great if the window re-sized automatically to make everything visible.
- It would also be good if it would be possible to re-size uBlock manually with mouse drag.

**Comment by seanrand** [comment]
How often does the average user take screenshots of the uBlock popup?
Where would you put the size limit? At 100% window height? Then you'd still have to scroll the list if it's longer.
Why not simply copy&paste the rules from the dynamic filtering tab in the settings instead?

(By the way, I'm not trying to just shoot down all your suggestions here, David. I'm just not sure if they're actually improvements.)

**Comment by davidhedlund** [comment]
@seanrand I think case I think you are right. I completely forgot about the "My rule" tab.

`(By the way, I'm not trying to just shoot down all your suggestions here, David. I'm just not sure if they're actually improvements.)`

I started this issue (based on a screenshot) https://github.com/chrisaljoudi/uBlock/issues/1371 which became adopted. Seems like the screenshot habbit kicked in from that issue. However, I have now replaced all screenshots with syntax when possible in all Issues I've started.
